apps/arweave/include/ar_blacklist_middleware.hrl

```diff
@@ -44,10 +44,14 @@ end).
 			{get_vdf_session, maps:get(get_vdf_session, LimitByIP, 120)};
 		[<<"vdf3">>, <<"session">>] ->
 			{get_vdf_session, maps:get(get_vdf_session, LimitByIP, 120)};
+		[<<"vdf4">>, <<"session">>] ->
```

Comment:
is /vdf4 a replacement for /vdf3?

Since /vdf3 was only released as part of the pre-release, will be able to remove it once "everyone" has upgraded to the final 2.7.4 release?

Reply:
yeah I was trying to think through it. I think your current approach is easiest (vdf3 + vdff4). We'll just try to remember to remove vdf3 sometime in the future...